torch/_functorch/functional_call.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Dict, Union, Any, Tuple
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+
+def functional_call(
+    module: 'torch.nn.Module',
+    parameter_and_buffer_dicts: Union[Dict[str, Tensor], Tuple[Dict[str, Tensor]]],
```

Comment:
Tuple[Dict[str, Tensor]], Dict[str, Tensor]]] to demonstrate it's a tuple of two things if that's what was intended